### ISSUE TITLE
PHP: Call to a member function getField() on a non-object

### ISSUE THREAD

**Issue by cyanwish** [google/protobuf#2433]
my proto file:
```
syntax = "proto3";

message Coordinates {
    sint32 lat = 1;
    sint32 lon = 2;
}
```
my code:

```
require_once __DIR__ . '/vendor/autoload.php';
require_once 'protobuf/Coordinates.pb.php';

$x = new Coordinates();
```
PHP 5.5.38 (homebrew), MacOSX El Capitan

I'm getting the following errors:
```
Notice: Undefined index: Coordinates in /Users/robert/PhpstormProjects/test_script/vendor/google/protobuf/php/src/Google/Protobuf/Internal/DescriptorPool.php on line 109

Call Stack:
PHP Notice:  Undefined index: Coordinates in /Users/robert/PhpstormProjects/test_script/vendor/google/protobuf/php/src/Google/Protobuf/Internal/DescriptorPool.php on line 109
    0.0491     492152   1. {main}() /Users/robert/PhpstormProjects/test_script/script.php:0
    0.1427    2026768   2. Google\Protobuf\Internal\Message->__construct(???) /Users/robert/PhpstormProjects/test_script/script.php:21
PHP Stack trace:
    0.1427    2026896   3. Google\Protobuf\Internal\DescriptorPool->getDescriptorByClassName(???) /Users/robert/PhpstormProjects/test_script/vendor/google/protobuf/php/src/Google/Protobuf/Internal/Message.php:74


Fatal error: Call to a member function getField() on a non-object in /Users/robert/PhpstormProjects/test_script/vendor/google/protobuf/php/src/Google/Protobuf/Internal/Message.php on line 75
PHP   1. {main}() /Users/robert/PhpstormProjects/test_script/script.php:0
PHP   2. Google\Protobuf\Internal\Message->__construct($desc = *uninitialized*) /Users/robert/PhpstormProjects/test_script/script.php:21
PHP   3. Google\Protobuf\Internal\DescriptorPool->getDescriptorByClassName($klass = *uninitialized*) /Users/robert/PhpstormProjects/test_script/vendor/google/protobuf/php/src/Google/Protobuf/Internal/Message.php:74

PHP Fatal error:  Call to a member function getField() on a non-object in /Users/robert/PhpstormProjects/test_script/vendor/google/protobuf/php/src/Google/Protobuf/Internal/Message.php on line 75
PHP Stack trace:
PHP   1. {main}() /Users/robert/PhpstormProjects/test_script/script.php:0
PHP   2. Google\Protobuf\Internal\Message->__construct($desc = *uninitialized*) /Users/robert/PhpstormProjects/test_script/script.php:21
Call Stack:
    0.0491     492152   1. {main}() /Users/robert/PhpstormProjects/test_script/script.php:0
    0.1427    2026768   2. Google\Protobuf\Internal\Message->__construct(???) /Users/robert/PhpstormProjects/test_script/script.php:21

Process finished with exit code 255
```
What am I doing wrong here?

**Comment by TeBoring** [comment]
Could you test against master head?
This occurred when there is no namespace, which should have been fixed.

**Comment by cyanwish** [comment]
master branch worked, but ```dev-master``` didnt include the fix, see #2440
